pkg/pillar/types/diskmetrics.go

```diff
@@ -46,13 +46,13 @@ var ReportDirPaths = []string{
 
 // AppPersistPaths  Application-related files live here
 // XXX do we need to exclude the ContentTrees used for eve image update?
```

Comment:
Good question... Does the comment still make sense?

Reply:
It does, but we don't have a good way to tell the CT which contains an EVE image from a CT which contains an app image.
If we want to better separate the device management from the app/workload managament, then we should revisit this.